addons/account/models/account_move.py

```diff
@@ -2105,6 +2111,29 @@ def _validate_taxes_country(self):
     # -------------------------------------------------------------------------
     # LOW-LEVEL METHODS
     # -------------------------------------------------------------------------
+    def _get_debit_from_balance(self, balance):
+        compare_res = self.env.company.currency_id.compare_amounts(balance, 0.0)
+        if self.is_storno:
+            return balance if compare_res < 0.0 else 0.0
+        return balance if compare_res > 0.0 else 0.0
+
+    def _get_credit_from_balance(self, balance):
+        compare_res = self.env.company.currency_id.compare_amounts(balance, 0.0)
+        if self.is_storno:
+            return -balance if compare_res > 0.0 else 0.0
+        return -balance if compare_res < 0.0 else 0.0
+
+    def _get_debit_from_line_balance(self, balance, line_balance):
```

Comment:
What's the difference between `_get_debit_from_balance` and `_get_debit_from_line_balance`?
Is the `balance` the total of the move? Then it should be `amount_total` or something like that. 

I also don't see where it is used